routes/user.js

```diff
@@ -20,6 +20,7 @@ router.use('/:id/reviews', reviewRouter)
 router.get('/', asyncWrapper(userController.getUsers))
 router.get('/:id', asyncWrapper(userController.getOneUser))
 router.get('/my-profile', setCurrentUserIdAndRole, asyncWrapper(userController.getOneUser))
+router.patch('/update-my-profile', setCurrentUserIdAndRole, asyncWrapper(userController.updateUserProfile))
```

Comment:
there will be more endpoints that are connected to a current user somehow, so prolly its better to group them accordingly? i dunnu, like to make an empty row space after 21 line, or move all those my-profile above all?

Reply:
yeap, i think that adding space after 21 line will be good and how do you think about idea to change name of "update-my-profile" on "my-profile" but method PATCH?